pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -344,7 +350,7 @@ func getSupportedSubsystems() map[subsystem]bool {
 // We would like to skip setting any values on the device cgroup in this case
 // but this is not possible with libcontainers Set() method
 // See https://github.com/opencontainers/runc/issues/932
-func setSupportedSubsystems(cgroupConfig *libcontainerconfigs.Cgroup) error {
+func setSupportedSubsystems(cgroupConfig *libcontainerconfigs.Cgroup, cgroupv2 bool) error {
```

Comment:
This argument doesn't appear to be used anywhere.

Reply:
thanks, dropped now